sdk/src/beta9/abstractions/function.py

```diff
@@ -79,7 +87,7 @@ def __init__(
         image: Image = Image(),
         timeout: int = 3600,
         retries: int = 3,
-        callback_url: Optional[str] = "",
+        callback_url: str = "",
```

Comment:
why remove the Optional type here?

Reply:
Hm, yeah upstream it defaults to None. When I see Optional but the default is "", it tells me something we got the Optional wrong. I'll change this to default to None.